exporter/exporterhelper/internal/metadata/generated_telemetry.go

```diff
@@ -51,39 +51,39 @@ func (tbof telemetryBuilderOptionFunc) apply(mb *TelemetryBuilder) {
 }
 
 // InitExporterQueueCapacity configures the ExporterQueueCapacity metric.
-func (builder *TelemetryBuilder) InitExporterQueueCapacity(cb func() int64, opts ...metric.ObserveOption) error {
+func (builder *TelemetryBuilder) InitExporterQueueCapacity(cb func() int64, opts ...metric.ObserveOption) (metric.Registration, error) {
```

Comment:
This file is generated. It shouldn't be modified manually (that's documented at its top).
This file needs to modified instead: https://github.com/open-telemetry/opentelemetry-collector/blob/main/cmd/mdatagen/internal/templates/telemetry.go.tmpl

Reply:
I see, this will mean other call sites will need to be modified to expect the two return values. Perhaps a good opportunity to verify for other possibilities for similar memory leaks.